index.d.ts

```diff
@@ -1,6 +1,6 @@
-import { SecretCallback, SecretCallbackLong } from 'express-jwt';
 import { Agent as HttpAgent } from 'http';
 import { Agent as HttpsAgent } from 'https';
+import type {Jwt, Secret} from 'jsonwebtoken'
```

Comment:
This change makes jwks-rsa only compatible with Typescript 3.8+ as the [type-only imports and exports](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html) feature was release with TS 3.8. For one of my projects, we are on TS 3.7.3, which means this breaks out compatability

Also - this line is missing a semicolon :)

Reply:
No problem - we just attempted to upgrade and ran into a couple of other issues. We will be putting up a pull request soon!